### ISSUE TITLE
Mask Loss

### ISSUE THREAD

**Issue by JihongJu** [broadinstitute/keras-rcnn#17]
Loss function for the mask branch of  [Mask R-CNN](https://arxiv.org/pdf/1703.06870.pdf)
